gravitee-apim-portal-webui-next/src/components/app-card/app-card.component.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input } from '@angular/core';
+import { MatButton } from '@angular/material/button';
+import { MatCard, MatCardActions, MatCardContent } from '@angular/material/card';
+import { RouterLink } from '@angular/router';
+
+import { PictureComponent } from '../picture/picture.component';
+
+@Component({
+  selector: 'app-card',
+  standalone: true,
+  imports: [MatButton, MatCard, MatCardActions, MatCardContent, PictureComponent, RouterLink],
+  templateUrl: './app-card.component.html',
+  styleUrl: './app-card.component.scss',
+})
+export class AppCardComponent {
```

Comment:
Not a huge fan of the name since it's confusing to have `ApplicationCardComponent` that exists as well. What about `TileCardComponent` or `CardListComponent` or ??

Reply:
Done